packages/codemods/src/transforms/v3/web-react/__testfixtures__/button-isSquare-prop-name.input.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
**question (if-minor):** Why is this added?

Reply:
i am getting errors everywhere: `'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.`

Reply:
ESLint problem? I think we want to ignore the fixtures or have agreed to do so, haven't we? If so, I can solve this separately.